packages/core/src/config/config.ts

```diff
@@ -1064,8 +1085,22 @@ export class Config {
     this.fileSystemService = fileSystemService;
   }
 
-  getCompressionThreshold(): number | undefined {
-    return this.compressionThreshold;
+  async getCompressionThreshold(): Promise<number | undefined> {
+    if (this.compressionThreshold) {
+      return this.compressionThreshold;
+    }
+
+    if (this.experimentsPromise) {
```

Comment:
need a try catch for this await ignoring the result  otherwise a failure to fetch the experiments will break calling getCompressionThreshold